Project/dataplot.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
I will add more description on functionality and things to do which should address a number of concerns.

I do have unused imports but I anticipate using them in the future as I expand the plotting and server options.